### PR TITLE
First pass at adding Segment find-by-pid pages

### PR DESCRIPTION
**Pull request by BPScott**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6309

Adds the details section at the top of the page, and requests all data
required in the Controller.

We still need to format the output of programmes the Segement appeared
on, and work out how to deal with the case when a Segment appears on
multiple programmes within the same Network.

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
@felipeparaujo are you already doing this page?

File: templates/find_by_pid/segment.html.twig [hunk 1]
```diff
@@ -0,0 +1,55 @@
+{% extends 'base_ds2013.html.twig' %}
+
+{% block page_classes %}programmes-page text-base programmes-page--flush{% endblock %}
+{% block title %}{{ meta_context.titlePrefix() ? meta_context.titlePrefix() ~ ' - ' }}{{ segment.getTitle() }}{% endblock %}
+
+{% block body %}
+<div class="b-g-p">
+<div class="map__column br-box-secondary">
```
Comment (votemike) [inline templates/find_by_pid/segment.html.twig]: Why is this div on the same level as the one above? This cause a weird PR if someone comes to auto format this file at a later date.

File: templates/find_by_pid/segment.html.twig [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    <div class="grid-wrapper">
+        <div class="grid 2/3@bpw2 3/4@bpe">
+            <div class="island">
+                <h1>{{ segment.getTitle() }}</h1>
```
Comment (votemike) [inline templates/find_by_pid/segment.html.twig]: Will the branding not have an H1 in it?

File: templates/find_by_pid/segment.html.twig (hunk 1 above)
Comment (BPScott) [reply templates/find_by_pid/segment.html.twig]: Will Fix.

File: templates/find_by_pid/segment.html.twig (hunk 2 above)
Comment (BPScott) [reply templates/find_by_pid/segment.html.twig]: No, the branding has no headers tags in it - as just the programme title would be a poor heading for all pages that hang off the a findbypid page e.g. for :pid/episodes you'd want "Episode Guide" in the heading.

Brands and Episodes have their <h1> in their MAP panels.

**Comment by BPScott** [comment]
AFAIK @felipeparaujo is working on "on the episode page, show a list of segments on this episode", this PR is for "on the segment page, give me info about the segment and the programmes it appears on".

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by votemike** [comment]
Test me please

**Pull request merged**